src/components/link/link_display/link_display.ts

```diff
@@ -7,7 +7,7 @@ import { CellPopoverComponent, PopoverBuilders } from "../../../types/cell_popov
 import { css } from "../../helpers/css";
 import { Menu } from "../../menu/menu";
 
-const LINK_TOOLTIP_HEIGHT = 43;
+const LINK_TOOLTIP_HEIGHT = 32;
```

Comment:
This really highlight a problem we had before : this size was just an arbitrary size that was only used to know if we display the component above or below the cell, but had nothing to do with the actual size in the DOM.